Cabal-QuickCheck/Cabal-QuickCheck.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 2.2
 name:          Cabal-QuickCheck
-version:       3.14.0.0
+version:       3.14.2
```

Comment:
Do we need to care about the internal packages?

Reply:
```suggestion
version:       3.14.0.0
```